content/docs/guides/query-log.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Query Log
+sort_rank: 1
+---
+
+# Using the Prometheus query log
+
+Prometheus has the ability to log all the queries run by the engine to a log
+file. This guide demonstrates how to use that log file, which fields it does
+contain and provides advanced tips about how to operate the log file.
+
+The query log has been introduced in v2.16.0 and is disabled by default.
```

Comment:
This is very awkward. The version should only be mentioned in passing, and the enablement is what the rest is about.